src/main/java/calculation/CalculateCoordinatesInterpolationAzimuthLength.java

```diff
@@ -0,0 +1,92 @@
+package calculation;
+
+import model.Location;
+import model.LocationPoint;
+import org.gavaghan.geodesy.Ellipsoid;
+import org.gavaghan.geodesy.GeodeticCalculator;
+import org.gavaghan.geodesy.GlobalCoordinates;
+import utils.Utilities;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CalculateCoordinatesInterpolationAzimuthLength {
+  public List<Location> calculatePointsOnRoutePrototype(LocationPoint locationPoint) {
```

Comment:
I think you can make this method a static one, you don't really need to create an object of the class CalculateCoordinatesInterpolationAzimuthLength to execute this method